### PR TITLE
Fix README.md duplicated line

### PR DESCRIPTION
**Pull request by EG-easy**
## Why

Fix typo. 
There was duplicated example in README.md.

## How

*

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,7 +39,6 @@ no-unexpected-multiline
       openreachtech/newline-per-parameter: error
       openreachtech/no-else-if: error
       openreachtech/no-if-in-oneline: error
-      openreachtech/newline-per-parameter: error
```
Comment (StewEucen) [inline README.md:42]:
```
+ openreachtech/no-unexpected-multiline: error
```

**Review by StewEucen — CHANGES_REQUESTED** [review]
@EG-easy 

* Thank you :)
* Mistook to rewrite after copying.

File: README.md (hunk 1 above)
Comment (EG-easy) [reply README.md:42]: - Updated!

**Review by StewEucen — APPROVED** [review]
🍙

**Pull request merged**